### PR TITLE
Fix for #1775 + #1754 + other changes

### PR DESCRIPTION
**Pull request by TurboProgramming**
Fix for #1775, #1754 and other changes for enhancements. 

**This PR includes:**
- Asset additions
- String updates
- String additions

### CODE REVIEW

File: app/src/main/res/drawable/ic_feedback_black_24dp.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```
Comment (k0shk0sh) [inline app/src/main/res/drawable/ic_feedback_black_24dp.xml]: Please don't hardcode the color but use the attribute that is being used by all the other icons, find any icon and you'll get what I mean, also rename the file once u done.

File: app/src/main/res/drawable/ic_group_black_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```
Comment (k0shk0sh) [inline app/src/main/res/drawable/ic_group_black_24dp.xml]: Same here

File: app/src/main/res/drawable/ic_person_black_24dp.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```
Comment (k0shk0sh) [inline app/src/main/res/drawable/ic_person_black_24dp.xml]: Same here

File: app/src/main/res/drawable/ic_code_black_24dp.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```
Comment (k0shk0sh) [inline app/src/main/res/drawable/ic_code_black_24dp.xml]: Same here

File: app/src/main/res/drawable/ic_feedback_black_24dp.xml (hunk 1 above)
Comment (TurboProgramming) [reply app/src/main/res/drawable/ic_feedback_black_24dp.xml]: I undertstood the renaming part, I'll rename the files as the ones you're already using, but I didn't understand what I need to change. Can you be more specific? or show me which lines I should change?

File: app/src/main/res/drawable/ic_code_black_24dp.xml (hunk 4 above)
Comment (TurboProgramming) [reply app/src/main/res/drawable/ic_code_black_24dp.xml]: Fixed.

File: app/src/main/res/drawable/ic_person_black_24dp.xml (hunk 3 above)
Comment (TurboProgramming) [reply app/src/main/res/drawable/ic_person_black_24dp.xml]: Fixed.

File: app/src/main/res/drawable/ic_group_black_24dp.xml (hunk 2 above)
Comment (TurboProgramming) [reply app/src/main/res/drawable/ic_group_black_24dp.xml]: Fixed.

File: app/src/main/res/drawable/ic_feedback_black_24dp.xml (hunk 1 above)
Comment (TurboProgramming) [reply app/src/main/res/drawable/ic_feedback_black_24dp.xml]: Fixed.

**Comment by TurboProgramming** [comment]
I've applied all changes you requested. Can you please check again if I did good?

File: app/src/main/res/menu/drawer_menu.xml [hunk 5]
```diff
@@ -17,11 +17,11 @@
     <group android:id="@+id/profile_group">
         <item
             android:id="@+id/profile"
-            android:icon="@drawable/ic_profile"
+            android:icon="@drawable/ic_person_black_24dp"
```
Comment (k0shk0sh) [inline app/src/main/res/menu/drawer_menu.xml]: Here

File: app/src/main/res/menu/drawer_menu.xml [hunk 6]
```diff
@@ -17,11 +17,11 @@
...
             android:title="@string/profile"/>
         <item
             android:id="@+id/orgs"
-            android:icon="@drawable/ic_people"
+            android:icon="@drawable/ic_group_black_24dp"
```
Comment (k0shk0sh) [inline app/src/main/res/menu/drawer_menu.xml]: Here

File: app/src/main/res/menu/drawer_menu.xml [hunk 7]
```diff
@@ -44,7 +44,7 @@
             android:title="@string/trending"/>
         <item
             android:id="@+id/gists"
-            android:icon="@drawable/ic_gists"
```
Comment (k0shk0sh) [inline app/src/main/res/menu/drawer_menu.xml]: Here

File: app/src/main/res/menu/drawer_menu.xml [hunk 8]
```diff
@@ -53,8 +53,8 @@
         android:checkableBehavior="none">
         <item
             android:id="@+id/reportBug"
-            android:icon="@drawable/ic_bug"
```
Comment (k0shk0sh) [inline app/src/main/res/menu/drawer_menu.xml]: And here

**Comment by TurboProgramming** [comment]
I guess its done now?

File: app/src/main/res/menu/drawer_menu.xml (hunk 7 above)
Comment (TurboProgramming) [reply app/src/main/res/menu/drawer_menu.xml]: Changed that to ic_code.

File: app/src/main/res/menu/drawer_menu.xml (hunk 8 above)
Comment (TurboProgramming) [reply app/src/main/res/menu/drawer_menu.xml]: Changed that to ic_feedback.

**Pull request merged**